packages/neuron-ui/src/components/MultisigAddress/hooks.ts

```diff
@@ -26,90 +27,111 @@ export const useSearch = (clearSelected: () => void) => {
 
   const onSearch = useCallback(
     value => {
-      setSearchKeywords(value)
+      onFilterConfig(value)
       clearSelected()
     },
-    [setSearchKeywords, clearSelected]
+    [onFilterConfig, clearSelected]
   )
 
   const onClear = useCallback(() => {
     onSearch('')
   }, [onSearch])
-  return { keywords, onKeywordsChange, setKeywords, onSearch, searchKeywords, onClear }
+  return { keywords, onKeywordsChange, setKeywords, onSearch, onClear }
 }
 
 export const useConfigManage = ({ walletId, isMainnet }: { walletId: string; isMainnet: boolean }) => {
-  const [configs, setConfigs] = useState<MultisigConfig[]>([])
+  const [entities, setEntities] = useState<MultisigEntity[]>([])
   const saveConfig = useCallback(
-    ({ m, n, r, addresses, fullPayload }) => {
+    ({ m, n, r, addresses }: { m: number; n: number; r: number; addresses: string[] }) => {
       return saveMultisigConfig({
         m,
         n,
         r,
-        addresses,
-        fullPayload,
+        blake160s: addresses.map(v => addressToScript(v).args),
         walletId,
       }).then(res => {
         if (isSuccessResponse(res)) {
-          if (res.result) {
-            setConfigs(v => [res.result!, ...v])
-          }
+          setEntities(v => (res.result ? [res.result, ...v] : v))
         } else {
           throw new Error(typeof res.message === 'string' ? res.message : res.message.content)
         }
       })
     },
-    [walletId, setConfigs]
+    [walletId, setEntities]
   )
   useEffect(() => {
-    getMultisigConfig({
-      walletId,
-    }).then(res => {
-      if (isSuccessResponse(res)) {
-        setConfigs(res.result)
+    getMultisigConfig(walletId).then(res => {
+      if (isSuccessResponse(res) && res.result) {
+        setEntities(res.result)
       }
     })
-  }, [setConfigs, walletId])
+  }, [setEntities, walletId, isMainnet])
```

Comment:
Is `isMainnet` necessary to trigger `getMultisigConfig`? I think it just affects `address format` and `balance`, which can be handled in the renderer process.